src/site/html.rs

```diff
@@ -58,12 +58,13 @@ pub fn build(config: &Config, content: String) -> String {
     <meta property="og:type" content="website" />
     <meta property="og:title" content=&config.name />
     {social_meta}
+    <link rel="stylesheet" href="http://sarahghp.com/themes/axo-oranda.css"></link>
```

Comment:
Update to  `https://www.unpkg.com/browse/@axodotdev/fringe/themes/axo-oranda.css` once https://github.com/axodotdev/turbo-axo/pull/176 is merged and released.